company/team/index.md

```diff
@@ -149,6 +149,7 @@ To add yourself to this page, copy the following template, paste it at the end o
 
 ## Dan Adler (he/him)
 
+- First name is pronounced **dan** (rhymes with "can")
```

Comment:
```suggestion
- “Dan” rhymes with "can"
```


The part not in parenthesis doesn’t seem to have any information content

Reply:
Hm, agreed, but the thinking was that both of the pieces could have information content for other names (e.g. see how Noemi did her bullet on this). I suppose there's no need to maintain a super-consistent format here though?

Reply:
my personal opinion is that I think it is ok to not be super consistent here